components/Login.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <div className="font-bold w-[694px] h-[564px] m-10">
+      <form className=" bg-acm-black rounded-[57px] grid">
+        <div className="grid pt-3 w-[607px] m-auto">
+          <label htmlFor="email" className="text-white pl-4 pb-6 text-4xl">
+            email
+          </label>
+          <input
+            type="email"
+            id="email"
+            className="rounded-full h-[76px] text-3xl"
+            placeholder="netid@ucr.edu"
+            required
+          />
+        </div>
+        <div className="grid pt-3 w-[607px] m-auto">
+          <label htmlFor="password" className="text-white pl-4 pb-6 text-4xl">
```

Comment:
```suggestion
          <label htmlFor="password" className="text-acm-white pl-4 pb-6 text-4xl">
```
Use the ACM color palette, it is the same but in case we ever change our white in the future!